src/MOI/MOI_callbacks.jl

```diff
@@ -59,12 +62,15 @@ end
 # ==============================================================================
 
 function default_moi_callback(model::Optimizer)
+    
     return (cb_data) -> begin
         get_cb_solution(model, cb_data.model)
         if model.heuristic_callback !== nothing
             model.callback_state = CB_HEURISTIC
             # only allow one heuristic solution per LP optimal node
-            if Xpress.getintattrib(cb_data.model, Xpress.Lib.XPRS_CALLBACKCOUNT_OPTNODE) > 1
+
+            if Xpress.getintattrib(cb_data.model, Lib.XPRS_CALLBACKCOUNT_OPTNODE) > 1
+                #println("Attribute ",Lib.XPRS_CALLBACKCOUNT_OPTNODE," value recovered : ",Lib.XPRSgetintattrib(cb_data.model, Lib.XPRS_CALLBACKCOUNT_OPTNODE, _))
```

Comment:
These were not here before, could you remove them?

Reply:
I see only one of them in my code, weird